api/v1alpha1/perconaservermysql_types.go

```diff
@@ -46,8 +46,11 @@ type PerconaServerMySQLSpec struct {
 	PMM                   *PMMSpec         `json:"pmm,omitempty"`
 	Backup                *BackupSpec      `json:"backup,omitempty"`
 	Router                *MySQLRouterSpec `json:"router,omitempty"`
+	UpgradeOptions        UpgradeOptions   `json:"upgradeOptions,omitempty"`
 }
 
+const Version = "0.2.0"
```

Comment:
please put it in a separate package called "version" like our other operators

Reply:
3336703af098a0467efa581d3f03f9a2de4a4c82